docs/index.md

```diff
@@ -41,10 +54,47 @@ $ docker-machine -v
 machine version 0.1.0
 ```
 
+In order to run Docker commands on your machines without having to use SSH, make
+sure to install the Docker client as well, e.g.:
+
+```
+$ curl https://get.docker.com/builds/Darwin/x86_64/docker-latest > /usr/local/bin/docker
+```
+
+### Windows
+
+The current recommended way to install Docker Machine on Windows is to use
+[msysgit](https://msysgit.github.io/).  This will provide you with some programs
+that Docker Machine relies on such as `ssh`, as well as a functioning shell.
+
+When you have installed msysgit, start up the terminal prompt and run the
+following commands.  Here it is assumed that you are on a 64-bit Windows
+installation.  If you are on a 32-bit installation, please substitute "i386" for
+"x86_64" in the URLs mentioned.
+
+First, install the Docker client binary:
+
+```
+curl https://get.docker.com/builds/Windows/x86_64/docker-latest > /bin/docker
```

Comment:
__UPDATE__: I think it's just your server that msysgit doesn't like the cert Evan :)